typed-builder-macro/src/field_info.rs

```diff
@@ -345,6 +378,34 @@ impl ApplyMeta for SetterSettings {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct Strip {
+    pub fallback: Option<syn::Ident>,
+    span: Span,
+}
+
+impl Strip {
+    fn new(span: Span) -> Self {
+        Self { fallback: None, span }
+    }
+}
+
+impl ApplyMeta for Strip {
+    fn apply_meta(&mut self, expr: AttrArg) -> Result<(), Error> {
+        match expr.name().to_string().as_str() {
+            "fallback" => {
+                let ident: syn::Ident = expr.key_value().map(|kv| kv.parse_value())??;
+                self.fallback = Some(ident);
```

Comment:
Since [you were concerened](https://github.com/idanarye/rust-typed-builder/pull/150#discussion_r1727107584) about "multiple sub attributes with the same name" - this is the place to handle this. If we were using `apply_flag_to_field` here, it would have checked internally that `self.fallback.is_none()` and if not it would have thrown an error. You can:

1. Do the same here (without the helper) - just throw an error if `self.fallback.is_none()`.
2. Create a new helper method, similar to `apply_flag_to_field`, that does it internally.

I think the second option can get a bit complicated, and I'm not sure it's worth doing RN.

About your other concern - "empty sub attributes" - I don't consider that an issue.